farmerbot/internal/power.go

```diff
@@ -148,7 +148,15 @@ func calculateResourceUsage(nodes map[uint32]node) (uint64, uint64) {
 }
 
 func (f *FarmerBot) resourceUsageTooHigh(sub Substrate) error {
-	for nodeID, node := range f.nodes {
+	nodesKeys := make([]uint32, 0, len(f.nodes))
+	for k := range f.nodes {
+		nodesKeys = append(nodesKeys, k)
+	}
+
+	for i := 0; i < len(f.nodes); i++ {
+		nodeID := nodesKeys[i]
+		node := f.nodes[nodeID]
+
```

Comment:
Is there a reason this is done this way? I mean, I think this does the same job as before